binaries/cuprated/src/rpc/request/address_book.rs

```diff
@@ -2,44 +2,112 @@
 
 use std::convert::Infallible;
 
-use anyhow::Error;
+use anyhow::{anyhow, Error};
+use cuprate_pruning::PruningSeed;
+use cuprate_rpc_types::misc::{ConnectionInfo, Span};
 use tower::ServiceExt;
 
 use cuprate_helper::cast::usize_to_u64;
 use cuprate_p2p_core::{
     services::{AddressBookRequest, AddressBookResponse},
+    types::BanState,
     AddressBook, NetworkZone,
 };
 
+// FIXME: use `anyhow::Error` over `tower::BoxError` in address book.
```

Comment:
### TODO
I tried replacing `tower::BoxError` with `anyhow::Error`, it's pretty deeply connected within `cuprate_consensus` but I think it works, the only problem was that one of the functions used (I forget which, either a `tower` or `tokio` one) for starting a service (I think) depends directly on the `tower::BoxError` type.

For now this uses `.map_err(|e| anyhow!(e))` which means we're boxing the error twice.